documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -1056,6 +1056,12 @@ The following table describes how the connector maps binary and character large
 
 |===
 
+[NOTE]
+====
+Oracle only supplies column values for `CLOB`, `NCLOB`, and `BLOB` data types if they're explicitly set or changed in a SQL statement.
```

Comment:
How are semantics for a BLOB column that did get changed? Do we emit old and new value then?

Reply:
So if any of the LOB types are changed, the `after` will contain the new value however the `before` part of the change event will contain the `unavailable.value.placeholder` value.

Reply:
Ok, can you clarify this a bit more then. Something to the account of:

> This means that change events will never contain the value of an unchanged `CLOB`, `NCLOB`, and `BLOB` column, but a placeholder as defined by the connector configuration property, `unavailable.value.placeholder`. If the value of a `CLOB`, `NCLOB`, or `BLOB` column gets updated, the new value will be contained in the `after` part of the corresponding update change events, whereas the unailable value placeholder will be used in the `before`part.